build/DockerfileForAlpine

```diff
@@ -1,4 +1,4 @@
-FROM nginx:1.17.7-alpine
+FROM nginx:1.17.7-alpine AS base
```

Comment:
If we pass `-alpine` as a build arg we could have just one file. And also support the other official images `-perl` and `alpine-perl`.

Reply:
I assume you mean something like:

```dockerfile
ARG BASE_TAG=1.17.7

FROM nginx:$BASE_TAG AS base
```

I don't think this can work because alpine uses the apk package manager, whereas debian uses the apt package manager so the rest of the stage can't succeed.

Reply:
Oh yeah you're right, I missed that we're installing a package. Not sure if there's a way to combine the two files to avoid the repetitions then.